nautobot/core/testing/views.py

```diff
@@ -333,7 +334,21 @@ def test_create_object_with_permission(self):
                 # Verify ObjectChange creation
                 objectchanges = lookup.get_changes_for_model(instance)
                 self.assertEqual(len(objectchanges), 1)
+                # Assert that Created By table row is updated with the user that created the object
                 self.assertEqual(objectchanges[0].action, extras_choices.ObjectChangeActionChoices.ACTION_CREATE)
+                # Validate if detail view exists
+                validate = URLValidator()
+                detail_url = instance.get_absolute_url()
+                try:
+                    validate(detail_url)
+                except ValidationError:
+                    self.skipTest("Model does not have a detail view")
```

Comment:
This will report the whole `test_create_object_with_permission` as skipped, which is probably not what you want here. Same concern below.